### PR TITLE
User/navbar/auth test

### PR DESCRIPTION
**Pull request by Tomo-Furu**
Additions:
-Added admin login routes, resister routes, logout routes in web.php
-Added AdminLoginController.php file
-Added AdminResisterController.php
-Added Admin.php in Model folder
-Added routes in admin.blade.php
-Added UI in login-admin.blade.php(Create a draft)
-Added UI in resister.blade.php for admin(Create a draft)
-Added admin for guards & providers in auth.php
-Added USER_MODEL=App\Models\User , ADMIN_MODEL=App\Models\Admin in .env

Migrated:
-Migrated create_admins_table

Deleted:
-Deleted function adminLogin() from LoginController

### CODE REVIEW

File: app/Http/Controllers/Auth/AdminLoginController.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    public function login(Request $request)
+    {
+        $this->validateLogin($request);
+
+        // Request check if the request is for an admin user.
+        if (Auth::guard('admin')->attempt(
+            $request->only('email', 'password'),
+            $request->filled('remember')
+        )) {
+            // If successful login
```
Comment (kredo-school) [inline app/Http/Controllers/Auth/AdminLoginController.php]: Please remove commented codes

File: app/Http/Controllers/Auth/AdminLoginController.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            // return redirect()->intended($this->redirectPath());
```
Comment (kredo-school) [inline app/Http/Controllers/Auth/AdminLoginController.php]: Please remove commented codes

File: config/auth.php [hunk 3]
```diff
@@ -35,13 +35,28 @@
     |
     */
 
+    // 'guards' => [
+    //     'web' => [
+    //         'driver' => 'session',
```
Comment (kredo-school) [inline config/auth.php]: Please remove commented codes

File: config/auth.php [hunk 4]
```diff
@@ -59,16 +74,20 @@
     |
     */
 
+    // 'providers' => [
+    //     'users' => [
```
Comment (kredo-school) [inline config/auth.php]: Please remove commented codes

File: config/auth.php [hunk 5]
```diff
@@ -59,16 +74,20 @@
...
+    //         'driver' => 'eloquent',
```
Comment (kredo-school) [inline config/auth.php]: Please remove commented codes

File: config/auth.php [hunk 6]
```diff
@@ -59,16 +74,20 @@
...
-            'model' => env('AUTH_MODEL', App\Models\User::class),
+            'model' => env('USER_MODEL', App\Models\User::class),
+        ],
+        'admins' => [
+            'driver' => 'eloquent',
+            'model' => env('ADMIN_MODEL', App\Models\Admin::class),
         ],
-
-        // 'users' => [
-        //     'driver' => 'database',
```
Comment (kredo-school) [inline config/auth.php:69]: Please remove commented codes

**Review by joacquinc — APPROVED** [review]

**Pull request merged**